benchmark/syclblas/blas2/tpmv.cpp

```diff
@@ -49,15 +51,23 @@ void run(benchmark::State& state, blas::SB_Handle* sb_handle_ptr,
       blas_benchmark::utils::Level2Op::tpmv, scalar_t>(state, "n", 0, 0, n);
 
   blas::SB_Handle& sb_handle = *sb_handle_ptr;
+  auto q = sb_handle.get_queue();
 
   // Input matrix/vector, output vector.
   std::vector<scalar_t> m_a =
       blas_benchmark::utils::random_data<scalar_t>(((n + 1) * n) / 2);
   std::vector<scalar_t> v_x =
       blas_benchmark::utils::random_data<scalar_t>(xlen);
 
-  auto m_a_gpu = blas::make_sycl_iterator_buffer<scalar_t>(m_a, m_a.size());
-  auto v_x_gpu = blas::make_sycl_iterator_buffer<scalar_t>(v_x, xlen);
+  auto m_a_gpu = blas::helper::allocate<mem_alloc, scalar_t>(m_a.size(), q);
+  auto v_x_gpu = blas::helper::allocate<mem_alloc, scalar_t>(xlen, q);
```

Comment:
Thanks for the TPMV USM extension! 
This is just a minor comment regarding variables affected by `incX` such as the size of `v_x`, it might be relevant to include it in the formulation of the size even-though it's hard-coded to 1, in order to avoid eventual confusion later if smth changes.

Reply:
Hi @OuadiElfarouki 
I have addressed this in [6417e3a363](https://github.com/codeplaysoftware/sycl-blas/pull/433/commits/6417e3a363ee90073af935a40e9a1c66c3a7e07a).